Documentation/network/egress-gateway.rst

```diff
@@ -76,6 +76,9 @@ Because egress gateway isn't compatible with identity allocation mode ``kvstore`
 you must use Kubernetes as Cilium's identity store (``identityAllocationMode``
 set to ``crd``). This is the default setting for new installations.
 
+Egress gateway is not compatible with Clustermesh. The gateway selected by an
```

Comment:
We refer to clustermesh a couple of different ways in the docs, but I believe the most common is `Cluster Mesh`.

```suggestion
Egress gateway is not compatible with Cluster Mesh. The gateway selected by an
```

Reply:
:+1: , fixed up.

@cilium/sig-clustermesh maybe y'all want to harmonize the remaining occurrences in the Docs?